assets/js/src/form_editor/components/notices.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { NoticeList } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+export default () => {
+  const dismissibleNotices = useSelect(
+    (select) => select('mailpoet-form-editor').getNotices(true),
```

Comment:
I don't think using boolean argument here is a good idea. From here I don't know what it does. Why don't you create two functions `getDismissibleNotices` and `getAllNotices`?